docs/develop/Applications.md

```diff
@@ -0,0 +1,67 @@
+
+### Ballot Encryption
+
+Ballot encryption is the process of converting voter selections into data such that it cannot be read by unauthorized parties. A ballot encryption app provides this fundamental functionality. The app functions as a tool that encrypts a plaintext ballot into a encrypted ballot aka ciphertext ballot. 
+
+Two quick ways to start are by using the ElectionGuard [encryption nuget package][encryption nuget package] or or [python package][python package].
+
+### Ballot Box
+
+A ballot box app takes the encrypted ballot of a voter and allows a user to submit the ballot as cast or spoil/challenge the ballot. A cast submitted ballot is aggregated into the tally. A spoiled submitted ballot is added to the list of spoiled ballots. 
+
+### Administer Election
+
+Administering an election for ElectionGuard usually requires the following steps. An example of this is available in [electionguard-ui repository][admin app].
+
+1. Setup Election - An election should be setup with a manifest and selected guardians
+2. Key Ceremony - A key ceremony should be run that results in the encryption key that can be used for the ballots.
+3. Tally Ceremony - A tally ceremony when run will decrypt the tally and any challenge ballots
+4. Election Record - An election record should be retrievable at the end of the election for publication and verification purposes. 
+
+A quick way to start is to use the [electionguard api-client][api client] and the [electionguard-python-api][api].
+
+### Publish Election
+
+The results of election can be displayed after an election is closed. An example of this is available in [electionguard-ui repository][result app]. This application should contain three pieces. 
+
+1. **Results** - Using decrypted tally and manifest, the tallied results should be displayed. 
+2. **Ballot Confirmation** - Confirm a ballot is in the Election Record.
+    - Cast Ballot - Confirm a ballot has been cast
+    - Challenge Ballot - Confirm a ballot has been challenged and display decrypted ballot
+3. **Download Election Record** - The Election Record should be downloadable for the public. 
+
+### Verify Election
+
+A verifier app verifies an election is true and accurate by using the publicly published information about a completed election. At the close of an election using ElectionGuard, an election record should be created and made public. ElectionGuard encourages third parties to create their own verifiers and help verify election records. A basic example of a verifier can be found within the [electionguard-python repository][verifier], but  verifiers should aim to be as thorough as possible. After consuming the files in an election record, verifiers can validate everything from the key ceremony to the decryption since the election is end to end verifiable. 
```

Comment:
This I'm going to leave. I agree with this but this is actually open ended depending on the organization. It could be the election jurisdiction, it could be the company running the election, it could be many people.

Reply:
Is there no term for this group of people within the ElectionGuard system?  "Verifiers" if nothing else?  What you just described is really useful to me and might have a place in the glossary, i.e.

```
### Verifiers

Verifiers are individuals that use a verifier app to ensure an election is true and accurate.  They are responsible for creating an election record and making it publicly available.  They could be many people depending on the individual election such as an election jurisdiction or the company running the election.
```